src/Lift.php

```diff
@@ -205,88 +204,73 @@ protected static function ignoredProperties(): array
                 }
 
-                $attributes = $reflectionProperty->getAttributes();
-
-                if (count($attributes) > 0) {
-                    $result[] = new PropertyInfo(
-                        name: $prop,
+                if ($attributes = $reflectionProperty->getAttributes()) {
+                    return new PropertyInfo(
+                        name: $propName,
                         value: $model->getAttribute($modelProp) ?? null,
                         attributes: collect($attributes),
                     );
                 }
-            } catch (ReflectionException) {
-                continue;
-            }
-        }
 
-        return collect($result);
+                return null;
+            })
+            ->filter();
     }
 
     private static function getMethodsWithAttributes(Model $model): Collection
     {
-        $publicMethods = self::getModelPublicMethods($model);
-        $result = [];
-
-        foreach ($publicMethods as $method) {
-            try {
-
-                $reflectionMethod = new ReflectionMethod($model, $method);
-                $attributes = $reflectionMethod->getAttributes();
-
-                if (count($attributes) > 0) {
-                    $result[] = new MethodInfo(
-                        name: $method,
-                        method: $reflectionMethod,
-                        attributes: collect($attributes),
-                    );
-                }
-            } catch (ReflectionException) {
-                continue;
-            }
-        }
-
-        return collect($result);
+        return collect(static::getModelPublicReflectionMethods($model))
+            ->filter(fn (ReflectionMethod $reflectionMethod): bool => (bool) $reflectionMethod->getAttributes())
+            ->map(fn (ReflectionMethod $reflectionMethod): MethodInfo => new MethodInfo(
+                name: $reflectionMethod->getName(),
+                method: $reflectionMethod,
+                attributes: collect($reflectionMethod->getAttributes()),
+            ));
     }
 
+    /**
+     * @return array<string>
+     */
     private static function getModelPublicProperties(Model $model): array
     {
-        $reflectionClass = new ReflectionClass($model);
-        $properties = [];
+        return array_column(static::getModelPublicReflectionProperties($model), 'name');
+    }
 
-        foreach ($reflectionClass->getProperties(ReflectionProperty::IS_PUBLIC) as $property) {
-            if (in_array($property->getName(), self::ignoredProperties())) {
-                continue;
-            }
-            $properties[] = $property->getName();
-        }
+    /**
+     * @return array<ReflectionProperty>
+     */
+    private static function getModelPublicReflectionProperties(Model $model): array
+    {
+        $properties = (new ReflectionClass($model))->getProperties(ReflectionProperty::IS_PUBLIC);
 
-        return $properties;
+        return array_values(array_diff_key(
```

Comment:
I don't like this line of code.
It's very confusing for newcomers because it combines a lot of calls in the same statement, please refactor this to be more clear and understandable.

Reply:
Added 2 additional variables to clarify